README.md

```diff
@@ -1,62 +1,54 @@
-# mig-operator
+# Konveyor Operator
 This operator will install velero with customized migration plugins, the migration controller, and migration UI used for migrating workloads from Openshift 3 to Openshift 4.
 
 ## Development
 See [hacking.md](./docs/hacking.md).
 
-## Operator Installation with OLM on Openshift 4
-1. `oc create -f mig-operator-source.yaml`
-1. Create a openshift-migration namespace
-1. In the left menu select Operator Hub and find `Migration Operator` in the list
-1. Click Install and install it in the mig namespace
-  * There are two channels to select from: latest and release-v1.
-  * latest roughly corresponds to beta.
-1. Once installation is complete select `Installed Operators` on the left menu
-1. Create a `MigrationController` CR. The default vales should be acceptable for 4.2
-1. For 4.1 add `deprecated_cors_configuration: true` under `spec:`
-1. The default `restic_timeout` is 1 hour, specified as `1h`. You can increase this if you anticipate doing large backups that will take longer than 1 hour so that your backups will succeed. The downside to increasing this value is that it may delay returning from unanticipated errors in some scenarios. Valid units are s, m, and h, which stand for second, minute, and hour.
+## Operator Installation on OpenShift 4 with OLM
+Konveyor Operator is available in Operator Hub. This is the recommended installation method for OpenShift 4.
+1. Browse or search for Konveyor Operator.
+1. Install the desired version.
+1. For pre-release versions see the development guide above.
 
-## Operator Installation without OLM
-The same channels are available for use without OLM. Do one of the following to install the desired version:
+## Operator Installation on OpenShift 3 without OLM
+The same versions are available for use without OLM. Run the command corresponding to the version you wish to use:
 
-`oc create -f deploy/non-olm/latest/operator.yml` 
-`oc create -f deploy/non-olm/v1.0/operator.yml` 
+`oc create -f deploy/non-olm/latest/operator.yml`  
+`oc create -f deploy/non-olm/v1.2/operator.yml`  
+`oc create -f deploy/non-olm/v1.1/operator.yml`  
 
-## Migration Controller Installation
-`controller-3.yml` and `controller-4.yml` in the `deploy/nom-olm/latest` and `v1.0` directories contain the recommended settings for OCP 3 and 4 respectively.
+## Component Installation and Configuration
+Component installation and configuration is accomplished by creating or modifying a MigrationController CR.
 
-Edit `controller-3.yml` or `controller-4.yml` and adjust options if desired.
+### Installing Components
```

Comment:
This bulleted list is very dense with info, a bit hard to parse IMO. 

Maybe it would make sense to split it out to a few sub sections, and maybe use paragraphs instead of bullets if we need to explain a lot of stuff.

**Possible sub-sections to split out for easier reading**
 - Versioned YAML manifests (discuss where to get YAML manifests for latest, 1.1, 1.2)
 - Recommended Install Topologies (discuss where components get installed)
 - Choosing components to install
 - ...

I'm not sure if this layout will fit perfectly with the content but think we could break this section out and benefit in readability gain.

Reply:
Does what is there now look/read any better/easier?